integration-tests/src/test/kotlin/org/stellar/anchor/platform/e2etest/Sep24End2EndTests.kt

```diff
@@ -310,8 +331,10 @@ class Sep24End2EndTests(config: TestConfig, val jwt: String) {
         .sign(keypair)
         .sign(newAcc)
 
-    wallet.stellar().submitTransaction(tx)
-
+    submissionLock.withLock {
+      wallet.stellar().submitTransaction(tx)
+      delay(5000)
```

Comment:
This is fine but I think maybe adding a retry would be more robust. We don't have to do this now though.

Reply:
The retry is already implemented: https://github.com/stellar/kotlin-wallet-sdk/blob/main/wallet-sdk/src/main/kotlin/org/stellar/walletsdk/horizon/Stellar.kt#L142